Core/Object Arts/Dolphin/ActiveX/OCX/AXControlSite.cls

```diff
@@ -795,6 +795,29 @@ state
 	sequence list addAll: controlState afterIndex: 1.
 	^sequence!
 
+storeOn: aStream indent: anInteger variables: aDictionary
+
+	| default name |
+	name := self storeNameIn: aDictionary.
+	aStream display: name; cr.
+	anInteger + 1 timesRepeat: [aStream tab].
```

Comment:
Is it worth bothering with attempting a poor-man's pretty print? You could cut some code and get a better quality result by just throwing this through the parser and formatter when you want to produce the final result.
Incidentally a general comment I have is that any formatting style is allowed, as long as it is the one produced by the RBFormatter as configured in the image :-) (to get this, Ctrl+Shift+S to reformat accept the methods, or Ctrl+W to reformat then without saving when working).